server/build.gradle

```diff
@@ -1,8 +1,9 @@
 plugins {
     id("com.github.johnrengelman.shadow") version "7.0.0"
-    id("io.micronaut.application") version "1.5.3"
+    id("io.micronaut.application") version "1.4.2"
```

Comment:
Was this downgrade intentional?

Reply:
It was unintentional - thanks for pointing that out. I reverted it